docs/docs/setup.md

```diff
@@ -40,15 +40,17 @@ Arrow Meta IDE Plugin is compatible with the same versions as [Kotlin Plugin for
 
 When using the Arrow Meta Gradle Plugin from Intellij IDEA, there is a Gradle task available to install the Arrow Meta IDE Plugin.
 
-This message will appear with every Gradle task execution if Arrow Meta IDE Plugin is not installed, for instance, when running a `clean` task:
+This message will appear with every Gradle task execution if Arrow Meta IDE Plugin isn't installed, for instance, when running a `clean` task:
 
 ```
 > Configure project :
 Arrow Meta IDE Plugin is not installed!
-Run 'install-idea-plugin' Gradle task under 'Arrow Meta' group to install it.
+Run 'install-idea-plugin' Gradle task under 'Arrow Meta' group to install it (choose just one project when multi-project)
```

Comment:
Nit: instead of 'Arrow Meta' I would say 'Tasks -> arrow meta'

Reply:
Changed, thanks @danimontoya :raised_hands: